pkg/cloud/api/type_trait.go

```diff
@@ -148,8 +148,12 @@ const (
 	// should never be set by the client.
 	FieldTypeOutputOnly FieldType = "OutputOnly"
 	// FieldTypeAllowZeroValue is an ordinary field that can be zero-value
```

Comment:
This does not make much sense now, as currently ordinary field can be a zero-value.

Reply:
I added TODO (line below) to remove this type when we migrate all resources. I just updated BackendService and HealthCheck in this PR. Remaining type I will do in next.